core/domain/services/admin/registrations/list_table/csv_reports/Registration.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace EventEspresso\core\domain\services\admin\registrations\list_table\csv_reports;
+
+use EE_Config;
+use EE_Error;
+use EE_Registry;
+use EEH_Export;
+use EEM_Registration;
+use ReflectionException;
+
+/**
+ * Class Registration
+ * Adds registration columns to the CSV row
+ *
+ * @author  Hossein Rafiei
+ * @package EventEspresso\core\domain\services\admin\registrations\list_table\csv_reports
+ */
+class Registration
+{
+
+    /**
+     * @var array
+     */
+    private $fields;
+
+
+    /**
+     * @var array
+     */
+    private $reg_row;
+
+
+    /**
+     * @var EEM_Registration
+     */
+    private $reg_model;
+
+
+    /**
+     * Registration constructor.
+     *
+     * @param array $fields
+     * @param array $reg_row
+     * @throws EE_Error
+     * @throws ReflectionException
+     */
+    public function __construct(array $fields, array $reg_row)
+    {
+        $this->fields = $fields;
+        $this->reg_row = $reg_row;
+        $this->reg_model = EE_Registry::instance()->load_model('Registration');
```

Comment:
the model singletons can be accessed directly like: `EEM_Registration::instance()`
and are always loaded on every request, so no need to call this loader method (it's super old)

Reply:
Thanks for sharing. With the new method (static functions), I receive it as parameter.